identity-enabler/holder-mobile-app/src/pages/Landing.svelte

```diff
@@ -94,7 +94,7 @@
         mounted = true;
 
         if (window.matchMedia("(pointer: coarse)").matches) {
-            const hammer = new Hammer(document.getElementById("wrapper"));
+            const hammer = new Hammer(document.querySelector("#wrapper"));
```

Comment:
why is this change needed?

Reply:
This rule: https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v40.1.0/docs/rules/prefer-query-selector.md